docs/LoaderApplicationInterface.md

```diff
@@ -222,6 +223,35 @@ dynamic library is named `libvulkan.dylib` and the ABI versioned library is
 currently named `libvulkan.1.dylib`.
 
 
+### Bundling the Loader With An Application
```

Comment:
Should this section mention the Vulkan Runtime installer that is available on Windows? It seems like that would be a less-bad way for Windows applications to ship a loader with their application (given that it won't overwrite the system installed loader if it's newer than the version provided with the runtime installer).

Reply:
Good point.  Thnx for the feedback.

Reply:
Updated.